src/bin/msbwt2-build.rs

```diff
@@ -42,9 +48,7 @@ fn main() {
 
     let fastx_fns: Vec<String> = matches.values_of_t("FASTX").unwrap_or_else(|_| vec![]);
     let out_fn: String = matches.value_of_t("out_bwt").unwrap_or_else(|_| "stdout".to_string());
-    //TODO: make this a command line option
-    let unsorted_strings: bool = matches.value_of_t("unsorted").unwrap_or(false);
-    let sorted_strings: bool = !unsorted_strings;
```

Comment:
The original way this was set-up was actually intentional.  The default _should_ be sorted and I was reluctant to enable the unsorted option.  I was potentially going to enable an override for this (hence the TODO) if and only if someone requested it.  Do you actually have a need for this or are you just filling in the TODO because you saw it listed as a possibility?